docs/apidocs/aer_quantum_info.rst

```diff
@@ -0,0 +1,6 @@
+.. _aer-quantum_info:
+
+.. automodule:: qiskit_aer.quantum_info.states
```

Comment:
The link name and the referenced module (and the file name) don't match up here.  I'm not sure how you want the documentation to be structured precisely, but it looks like there's a slight mistake either way.